lib/plugins/object.js

```diff
@@ -40,7 +40,7 @@ module.exports = fp(
         },
         TableName: OBJECTS_TABLE
       };
-      const { Item: item } = await dynamo.get(params).promise();
+      const { Item: item } = await dynamo.put(params).promise();
```

Comment:
Since you now `await` in this function, it should be `async`.

Reply:
Check again :)